app/native/webpack.main.config.js

```diff
@@ -3,9 +3,9 @@ module.exports = {
    * This is the main entry point for your application, it's the first file
    * that runs in the main process.
    */
-  entry: './src/main.js',
+  entry: './native/main.js',
```

Comment:
From what I've seen in the community, the convention is:
```
src/main
src/renderer
```
I know it's just a convention, but is there a reason to diverge from that?

Reply:
From where I stand our app is the functional core providing the feature set according to our product vision. Where electron is a deployment/container hosting/running the app. By separating like this it becomes simpler to understand responsibilities.

The core app interface here is a React component called `App`, which is a managable surface and could be further extended by using props for example. That in turn means that the any container needs to be able to run this React component.